sagemaker-clarify/time_series_byom/time_series_bring_your_own_model.ipynb

```diff
@@ -0,0 +1,1244 @@
+    "\n",
+    "**References:**\n",
+    "- Our main scientific reference is https://arxiv.org/abs/1910.06358. We scale the approach of the paper to include also static covariates, related time series and implement a stochastic estimator for efficiency.\n",
+    "- A very useful math reference is [Probabilistic values by RJ Weber](http://www.library.fa.ru/files/Roth2.pdf#page=109);  specifically, section 8 about random-order values (these are the same mathematical construction of ASV). "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "91f38cc4",
+   "metadata": {},
+   "source": [
+    "### Create `AsymmetricShapleyValueConfig`\n",
+    "\n",
+    "An `AsymmetricShapleyValueConfig` is used to configure the algorithm Clarify uses for time series explainability. It takes the following arguments:\n",
+    "\n",
+    "- `direction`: direction of explanation to be used. Available explanation types are `\"chronological\"`, `\"anti_chronological\"`, `\"bidirectional\"`. The cronological direction highlights the effect of older timesteps over more recent one, while the anti-chronological direction higlights the effect of timesteps closer to the forecasting. Bidirectional is a combination of the previous two modes. \n",
+    "- `granularity`: Granularity of explanation to be used. Available granularities are `\"timewise\"` and `\"fine_grained\"`. The first granularity is fast and computes the attribution of individual timesteps toward the forecast, not making distinctions of related time series. The fine-grained mode is slower, but computes an attribution for every timestep and every feature dynamic, distinguishing between related and target TS.\n",
+    "- `num_samples`: Number of samples to be used in the Asymmetric Shapley Value forecasting algorithm. Only applicable when using `\"fine_grained\"`  explanations. This represents the number of permutations sampled for computing the ASV. \n",
+    "- `baseline`: baseline configuration (dictionary). The baseline config is used to replace out-of-coalition values for the corresponding datasets (also known as background data). For temporal data (target time series, related time series), the baseline value types are `\"zero\"`, where all out-of-coalition values will be replaced with `0.0`, or `\"mean\"`, all out-of-coalition values will be replaced with the average of a time series. For static data(static covariates), a baseline value for each covariate should be provided for each possible item_id. An example config follows, where ``item1`` and ``item2`` are item ids::\n",
```

Comment:
Formatting for baseline isn't working here, can you format it as a code block? Just copy paste this in.

\`\`\` 
{
&emsp;"target_time_series": "zero",
&emsp;"related_time_series": "zero",
&emsp;"static_covariates":
&emsp;&emsp;"item1": [1, 1],
&emsp;&emsp;"item2": [0, 1],
&emsp;}
}
\`\`\`

Reply:
Make sure to do for both notebooks.

Reply:
addressed in the next revision